schemas/http.yml

```diff
@@ -0,0 +1,43 @@
+---
+- name: http
+  title: HTTP
+  group: 2
+  description: >
+    Fields related to HTTP requests and responses.
```

Comment:
A minor issue, but I'd suggest broadening the definition to something like.  Fields related to HTTP activity.  (Since, for example, `http.version` is not technically a request or a response.)

Otherwise, LGTM